core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java

```diff
@@ -1808,6 +1809,15 @@ public ServerBuilder http1HeaderNaming(Http1HeaderNaming http1HeaderNaming) {
         return this;
     }
 
+    /**
+     * Sets the duration between logging uncaught exceptions.
+     * @param exceptionReportingInterval - the duration between logging uncaught exceptions.
+     */
+    public ServerBuilder exceptionReportingInterval(Duration exceptionReportingInterval) {
+        this.exceptionReportingInterval = exceptionReportingInterval;
```

Comment:
Could we add some null check and normalization here? e.g.

- Use `requireNonNull` to throw a NPE.
- Normalize to `Duration#ZERO` if non-positive.

Reply:
All right ~ I'll add it.